src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +196,20 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        interpretInputAndExecuteTillExit();
+    }
+
```

Comment:
Ensure indentations are as per coding style manual after code changes. eclipse auto format makes this easy to do!